core/src/trezor/lvglui/scrs/address.py

```
@@ -499,7 +545,72 @@ def _prepare_nexa_message(self):
 
         msg_class = getattr(messages, self.current_chain_info["msg_class"])
         return msg_class(address_n=path, show_display=False, prefix="nexa")
+    
+    def _prepare_doge_message(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_prepare_doge_message`

(ANN202)